src/PowerShellEditorServices.VSCode/Cmdlets/VSCodeHtmlContentViewCommands.cs

```diff
@@ -0,0 +1,211 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Management.Automation;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.PowerShell.EditorServices.Engine.Services.PowerShellContext;
+using Microsoft.PowerShell.EditorServices.VSCode.CustomViews;
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
+
+namespace Microsoft.PowerShell.EditorServices.VSCode
+{
+    ///
+    [Cmdlet(VerbsCommon.New,"VSCodeHtmlContentView")]
+    [OutputType(typeof(IHtmlContentView))]
+    public class NewVSCodeHtmlContentViewCommand : PSCmdlet
+    {
+        private HtmlContentViewsFeature _htmlContentViewsFeature;
+        private ILogger _logger;
+
+        ///
+        [Parameter(Mandatory = true, Position = 0)]
+        [ValidateNotNullOrEmpty()]
+        public string Title { get; set; }
+
+        ///
+        [Parameter(Mandatory = false, Position = 1)]
+        public ViewColumn? ShowInColumn { get; set; }
+
+        ///
+        protected override void BeginProcessing()
+        {
+        }
+
+        ///
+        protected override void ProcessRecord()
+        {
+            if (_htmlContentViewsFeature == null)
+            {
+                if(this.GetVariableValue("psEditor") is EditorObject psEditor)
+                {
+                    _logger = psEditor.Components.GetService<ILoggerFactory>().CreateLogger("PowerShellEditorServices.VSCode");
+
+                    _htmlContentViewsFeature = new HtmlContentViewsFeature(
+                        psEditor.Components.GetService<ILanguageServer>(),
+                        _logger);
+
+                    _logger.LogInformation("PowerShell Editor Services VS Code module loaded.");
+                }
+                else
+                {
+                    throw new Exception("unable to fetch psEditor value");
+                }
+            }
+
+            IHtmlContentView view = _htmlContentViewsFeature.CreateHtmlContentViewAsync(Title).Result;
```

Comment:
Also in all instances of `GetAwaiter().GetResult()` or others, consider catching exceptions to rewrite them under `WriteError` or `ThrowTerminatingError`.  Preferably a targeted catch but a catch all is fine too.

Reply:
Added this.